distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+As most configurations are stored in configuration files, in some instances it may make sense to edit those configuration files directly.
+
+==== Configuring ${branding} Global Settings
+
+Global configuration settings are configured via the properties file `system.properties`.
```

Comment:
We should add a note saying that any changes made to this file require a restart of the system.